src/main/java/htsjdk/samtools/cram/io/CramInt.java

```diff
@@ -40,7 +40,10 @@ public static int int32(final byte[] data) {
      * @return an integer value read from the buffer
      */
     public static int int32(final ByteBuffer buffer) {
-        return buffer.get() | buffer.get() << 8 | buffer.get() << 16 | buffer.get() << 24;
```

Comment:
These shifts were being attempted in a `byte` context

Reply:
Do you have a test case that fails without this change?